modules/xbox360/xbox360_vcxproj.lua

```diff
@@ -0,0 +1,439 @@
+--
+-- xbox360/xbox360_vcxproj.lua
+-- Xbox 360 integration for vstudio.
+-- Copyright (c) 2012-2015 Manu Evans and the Premake project
```

Comment:
Related to first comment. Year and "main" copyright holder.

Reply:
I did indeed. Amended 👍